test/cloudflare.test.js

```diff
@@ -110,6 +266,13 @@ test('getDeploymentUrl() should fail if there are no matching builds', async ()
   axios.get.mockResolvedValueOnce(data)
 
   await expect(
-    getDeploymentUrl('123xyz', 'zentered', 'fix/huge-bug', 'zentered.co')
+    getDeploymentUrl(
+      '123xyz',
+      'zentered',
+      'fix/huge-bug',
+      'zentered.co',
+      'preview',
+      null
+    )
   ).rejects.toThrow()
```

Comment:
```suggestion
  ).rejects.toThrow('no matching builds found')
```

You discovered an issue with the tests here. Without `source` and `deployment_trigger`, there's another error that's being thrown, not the `"no matching builds"`. Could you change this test to include the missing attributes in the mock object?